packages/server/src/sdk/app/backups/backup.ts

```diff
@@ -0,0 +1,145 @@
+import { backups } from "@budibase/pro"
+import { db as dbCore, objectStore, tenancy } from "@budibase/backend-core"
+import {
+  AppBackupQueueData,
+  AppBackupStatus,
+  AppBackupTrigger,
+  AppBackupType,
+} from "@budibase/types"
+import { exportApp } from "./exports"
+import { importApp } from "./imports"
+import { calculateBackupStats } from "../statistics"
+import { Job } from "bull"
+import fs from "fs"
+import env from "../../../environment"
+
+type BackupOpts = {
+  doc?: { id: string; rev: string }
+  createdBy?: string
+}
+
+async function removeExistingApp(devId: string) {
+  const devDb = dbCore.dangerousGetDB(devId, { skip_setup: true })
+  await devDb.destroy()
+}
+
+async function runBackup(
```

Comment:
There's a lot of pro backup specific logic here, we could probably hide this all behind pro if we had a function for initialising the backups sdk like:

```ts
backups.init({
  importFn,
  exportFn,
  statisticsFn
})
```

This is similar to how we have the `init` function on backend core. We could make it generic so that we could say

```ts
pro.init({
   backups: {...} // as above
})
```


Pro could then setup the processors directly, and we could call `runBackup` directly from the consumer in app-service without this wrapper file.